kube-derive/src/custom_resource.rs

```diff
@@ -256,11 +256,14 @@ pub(crate) fn derive(input: proc_macro2::TokenStream) -> proc_macro2::TokenStrea
         #[serde(rename_all = "camelCase")]
         #visibility struct #rootident {
             #schemars_skip
+            /// Metadata on derived type
             #visibility metadata: #k8s_openapi::apimachinery::pkg::apis::meta::v1::ObjectMeta,
+            /// Spec on derived type
```

Comment:
ah, this is actually annoying... it means schemars will use this rather than what's in front of FooSpec:

```
    expected:
        "Spec on derived type"
    actual:
        "Our spec for Foo\n\nA struct with our chosen Kind will be created for us, using the following kube attrs"
```

via crd_derive tests. Probably need to try to re-use the docs for the type in here.